src/base/http/responsegenerator.h

```diff
@@ -37,11 +37,9 @@
 
 namespace Http
 {
-    class ResponseGenerator
-    {
-    public:
-        static QByteArray generate(Response response);
-    };
+    QByteArray responseGenerator(Response response);
```

Comment:
Why do you use these inappropriate method names? They usually should have the verb form.